src/11_6_2021_RcppClock_benchmarking_Rcpp_code.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Benchmarking Rcpp code with RcppClock"
+author: "Zach DeBruine"
+license: GPL (>= 2)
+tags: Rcpp benchmarking RcppClock
+summary: Rcpp benchmarking with a simple C++ tick-tock clock and seamless integration
+  with the R environment at runtime.
+---
+
+Rcpp is all about the need for speed. However, Rcpp code is not magically fast, but is the result of careful coding and profiling. RcppClock makes it easy to profile C++ code from R:

```

Comment:
We often typeset package names with either backticks or bold font (see R Journal).  Not all posts do, but consider doing it.

Reply:
all instances of R packages now in backticks (i.e. `RcppClock`)

Reply:
Nice. That works. I don't think I have a preferred style, really, but it is what I used most often.